hw2/03-form.html

```diff
@@ -12,8 +12,66 @@
     <script defer src="03-form.js"></script>
   </head>
   <body class="bg-dark">
-    <form class="bg-light border rounded w-50 mx-auto mt-5 p-3">
+    <form class="bg-light border rounded w-50 mx-auto mt-5 p-4">
       <h1 class="mt-2 mb-4">Contact Form</h1>
+      <div class="mb-3 form-group">
+        <label for="name" class="col-4 col-form-label">Name <sup>*</sup></label>
+        <input
+          type="text"
+          name="name"
+          id="name"
+          placeholder="caterina"
```

Comment:
Your program is fantastic, I love the tooltips about the email format. I feel like I should make that change to mine as well!

The only thing I want to mention is that the assignment says there should be no placeholder information in the form, but I see that you have some. Once you remove that, this program is perfection!